src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/questionWithResponses.tag

```diff
@@ -54,6 +56,20 @@
         </ul>
       </c:if>
 
+      <c:if test="${not isRecipientNameHidden}">
+        <div class="col-sm-12 form-inline mobile-align-left">
+          <label for="input"
+              style="text-indent: 24px">
+              <span data-toggle="tooltip"
+                  data-placement="top"
+                  title="<%= Const.Tooltips.EVALUEE_DESCRIPTION %>">
+                  Evaluee/Recipient
+              </span>
```

Comment:
This `<span>` block should be pushed to the left by 2 spaces. Additionally, the `Evaluee/Recipient` text.

Reply:
Okay